gui/default/syncthing/core/syncthingController.js

```diff
@@ -1661,6 +1661,20 @@ angular.module('syncthing.core')
             $http.post(urlbase + "/db/scan?folder=" + encodeURIComponent(folder));
         };
 
+        $scope.setAllFolderPause = function(pause) {
+            var folderListCache = $scope.folderList();
+
+            for (var i = 0; i < folderListCache.length; i++) {
+                var folder = folderListCache[i];
+                if (folder) {
```

Comment:
There should be no circumstances where an entry in the folderlist is null/undefined?